.claude/commands/address-pr-feedback.md

```
@@ -0,0 +1,242 @@
+echo "=== File Review Comments ==="
+gh api repos/$REPO/pulls/$PR_NUMBER/comments \
+  --jq '.[] | {id: .id, type: "review", path: .path, line: .line, body: .body[0:300], user: .user.login, in_reply_to_id: .in_reply_to_id}'
+
+# Fetch general PR comments (not tied to specific lines)
+echo "=== General PR Comments ==="
+gh api repos/$REPO/issues/$PR_NUMBER/comments \
+  --jq '.[] | {id: .id, type: "issue", body: .body[0:300], user: .user.login}'
+```
+
+**Comment Types:**
+| Type | API Endpoint | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)